Pod/Classes/VOKCoreDataManager.m

```diff
@@ -161,8 +145,7 @@ - (void)initPersistentStoreCoordinator
                               NSInferMappingModelAutomaticallyOption: @YES,
                               };
     
-    NSError *error;
-    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:[self managedObjectModel]];
+    _persistentStoreCoordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:self.managedObjectModel];
```

Comment:
:shrug: like I said, it's a matter of opinion
If nobody else dislikes the *sprawling* init methods then I won't mind.